File: app/src/main/java/com/vkpapps/soundbooster/interfaces/OnControlRequestListener.java
package com.vkpapps.soundbooster.interfaces;

import com.vkpapps.soundbooster.model.control.ControlPlayer;

public interface OnControlRequestListener {
    void onMusicPlayerControl(ControlPlayer controlPlayer);

    void onDownloadRequest(String name, String id);

    void onDownloadRequestAccepted(String name, String id);

    void onUploadRequest(String name, String id);

    void onUploadRequestAccepted(String name, String id);

}

File: app/src/main/java/com/vkpapps/soundbooster/interfaces/OnControlRequestListener.kt
package com.vkpapps.soundbooster.interfaces

import com.vkpapps.soundbooster.model.control.ControlPlayer

/**
 * @author VIJAY PATIDAR
 */
interface OnControlRequestListener {
    fun onMusicPlayerControl(controlPlayer: ControlPlayer)
    fun onDownloadRequest(name: String, id: String, type: Int)
    fun onDownloadRequestAccepted(name: String, id: String, type: Int)
    fun onUploadRequest(name: String, id: String, type: Int)
    fun onUploadRequestAccepted(name: String, id: String, type: Int)
}